survey-sparrow-android-sdk/src/main/java/com/surveysparrow/ss_android_sdk/SsSurveyFragment.java

```diff
@@ -158,20 +203,52 @@ public void onResponse(String response) {
         ssWebView.getSettings().setDomStorageEnabled(true);
         ssWebView.addJavascriptInterface(new JsObject(), "SsAndroidSdk");
 
+        ConstraintLayout constraintLayout = new ConstraintLayout(getActivity());
+        constraintLayout.setLayoutParams(new ConstraintLayout.LayoutParams(
+                ConstraintLayout.LayoutParams.MATCH_PARENT,
+                ConstraintLayout.LayoutParams.MATCH_PARENT
+        ));
+
         final ImageButton closeButton = new ImageButton(getActivity());
         closeButton.setImageResource(R.drawable.ic_close_black);
         closeButton.setBackgroundResource(R.drawable.ic_close_rounded);
         closeButton.setClickable(true);
         closeButton.setPadding(10, 10, 10, 10);
         closeButton.setVisibility(View.GONE);
 
-        FrameLayout.LayoutParams closeButtonParams = new FrameLayout.LayoutParams(
-            FrameLayout.LayoutParams.WRAP_CONTENT,
-            FrameLayout.LayoutParams.WRAP_CONTENT
+        ConstraintLayout.LayoutParams closeButtonParams = new ConstraintLayout.LayoutParams(
+                ConstraintLayout.LayoutParams.WRAP_CONTENT,
+                ConstraintLayout.LayoutParams.WRAP_CONTENT
         );
-        closeButtonParams.setMargins(0, 60, 60, 0);
-        closeButtonParams.gravity = Gravity.TOP | Gravity.END;
+
+        int marginInDp = 20;
+        int marginInPx = (int) TypedValue.applyDimension(
+                TypedValue.COMPLEX_UNIT_DIP, marginInDp, getResources().getDisplayMetrics()
+        );
+
+        closeButtonParams.endToEnd = ConstraintLayout.LayoutParams.PARENT_ID;
+        closeButtonParams.topToTop = ConstraintLayout.LayoutParams.PARENT_ID;
+        closeButtonParams.setMargins(0, marginInPx, marginInPx, 0);
+
         closeButton.setLayoutParams(closeButtonParams);
+        constraintLayout.addView(closeButton);
+
+
+        HashMap properties = survey.getProperties();
+
+
+
+
+
+
+        ssWebView.setWebViewClient(new WebViewClient() {
```

Comment:
@KalaiselvanK-018 

ssWebView.setWebViewClient(new WebViewClient() {
            @Override
            public void onPageFinished(WebView view, String url) {
                super.onPageFinished(view, url);
                String jsCode = "const styleTag = document.createElement('style'); styleTag.innerHTML = `.ss-language-selector--wrapper { margin-right: 45px; }`; document.body.appendChild(styleTag);";
                view.evaluateJavascript(jsCode, null);
            }
        });


this code seems to be duplicate, can we check it ?